pelican/plugins/seo/seo.py

```diff
@@ -43,28 +36,67 @@ def plugin_initializer(settings):
     logger.info("SEO plugin initialized")
 
 
+def get_plugin_settings(context):
+    """Get settings in the pelican config file from the given context
+
+    .. note:: Pelican's settings take precedence over the plugin's default settings.
+
+    SEO report settings:
+
+        - SEO_REPORT
+        - SEO_ARTICLES_LIMIT
+        - SEO_PAGES_LIMIT
+
+    SEO enhancer settings:
+
+        - SEO_ENHANCER
+        - SEO_ENHANCER_OPEN_GRAPH
+        - SEO_ENHANCER_TWITTER_CARDS
+
+    :return: Dictionary of settings for the plugin
+    :rtype: <dict>
+    """
+
+    def get_seo_variables(data):
+        """Get variables starting with 'SEO_' & values in the given dictionary
+        :rtype: <dict>
+        """
+        return {
+            var_name: value
+            for var_name, value in data.items()
```

Comment:
```suggestion
            for setting_name, setting_value in settings.items()
```
I prefer explicit variable names :)

Reply:
Indeed you are right my `data` variable is bad, I propose `name, value in settings.items()`
(No unnecessary redundancy of the term settings)

Reply:
I will rename the import `from . import settings as default_plugin_settings` to avoid the shadowing of outer scope `settings`

Reply:
Ok for me